### ISSUE TITLE
City Results Screen Not Able to Scroll Results

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Closure-App#15]
After searching for a city by typing in "san" I get results but can't scroll through the whole list to be able to get to "San Marcos, TX" which is farther down the list and not visible on the screen.

See attached screenshot.

![closure notification - not able to scroll search results - looking for san marcos tx](https://user-images.githubusercontent.com/10423487/36498912-c809a520-170d-11e8-876f-7f23cd376501.png)

**Comment by jfc3-dol** [comment]
Make sure to check for the same issue in both the "By City" and "By State" tabs.
